### PR TITLE
Keepers Job Scheduler 

### PR DESCRIPTION
**Pull request by ritika-b**
- adds keepers job scheduler
- updates nav
- add tutorial

### CODE REVIEW

File: docs/chainlink-keepers/job-scheduler.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+**Table of Contents**
++ [Overview](#overview)
++ [Register a new Upkeep](#register-a-new-upkeep)
++ [Trigger Selection](#trigger-selection)
++ [Using Time-Based Triggers](#using-time-based-triggers)
++ [Using Custom Logic Triggers](#using-custom-logic-triggers)
+
+# Overview
+
+Learn how to register a contract with the Chainlink Keepers network and automate it using the Keepers job scheduler. We encourage you to learn how to [create a Keeper-compatible contract](../compatible-contracts) before using this guide.
```
Comment (dwightjl) [inline docs/chainlink-keepers/job-scheduler.md:21]:
```suggestion

Learn how to register a contract with the Chainlink Keepers network and automate it using the Keepers job scheduler. Before you begin, learn how to [create a Keeper-compatible contract](../compatible-contracts).

**Table of Contents**

+ [Register a new Upkeep](#register-a-new-upkeep)
+ [Trigger Selection](#trigger-selection)
+ [Using Time-Based Triggers](#using-time-based-triggers)
+ [Using Custom Logic Triggers](#using-custom-logic-triggers)
```
Swap the overview with the TOC and remove the overview header.

File: docs/chainlink-keepers/job-scheduler.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Connecting your Wallet
+
+If you do not already have a wallet connected with the Keepers network, the interface will prompt you to do so. Click the **Connect Wallet** button and follow the remaining prompts to connect your wallet to the network.
+
+![Keepers Connect Waller](/images/contract-devs/keeper/keeper-connect-wallet.png)
+
+# Trigger Selection
+
+Once you have successfully connected your wallet, you will have options to create a trigger mechanism for automation. The trigger specifies what Keeper Nodes should look at to determine if your Upkeep should be performed. The page will present two options: [*time-based*](#using-time-based-triggers) and [*custom logic*](#using-custom-logic-triggers) triggers.  Time-based uses a time schedule (CRON) to execute your smart contract function according to the schedule. Custom logic uses a Keeper-compatible contract that you deployed to determine when to perform your Upkeep. To learn more about Keeper-compatible contracts, refer to our [guide](../compatible-contracts).
```
Comment (dwightjl) [inline docs/chainlink-keepers/job-scheduler.md:37]:
```suggestion
Once you have successfully connected your wallet, you will have options to create a trigger mechanism for automation. The trigger specifies what Keeper Nodes should look at to determine if your Upkeep should be performed. The page presents two options:

- [Time-based triggers](#using-time-based-triggers): Uses a time schedule (CRON) to execute your smart contract function according to the schedule
- [Custom logic triggers](#using-custom-logic-triggers): Uses a Keeper-compatible contract that you deployed to determine when to perform your Upkeep.

To learn how to create Keeper-compatible contracts, see [Making Keepers-compatible Contracts](../compatible-contracts).
```

File: docs/chainlink-keepers/job-scheduler.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+![Keepers Trigger Selection](/images/contract-devs/keeper/keeper-trigger-selection.png)
+
+# Using Time-Based Triggers
+
+To use the time-based trigger, you will be prompted to enter a *contract address*. This is the address of the deployed contract that contains the function that you want Keepers to execute. If you did not verify the contract on chain, you will also need to paste the [Application Binary Interface](https://docs.soliditylang.org/en/develop/abi-spec.html) (ABI) of the deployed contract into the corresponding text box.
```
Comment (dwightjl) [inline docs/chainlink-keepers/job-scheduler.md:43]:
```suggestion
When you select the time-based trigger, you are prompted to enter a *contract address*. This is the address of the deployed contract that contains the function that you want Keepers to execute. If you did not verify the contract on chain, you will also need to paste the [Application Binary Interface](https://docs.soliditylang.org/en/develop/abi-spec.html) (ABI) of the deployed contract into the corresponding text box.
```

Present tense

File: docs/chainlink-keepers/job-scheduler.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+![Keepers Time Based Trigger](/images/contract-devs/keeper/keeper-time-based-trigger.png)
+
+## Specifying the Time Schedule
+
+Once you have successfully entered your contract address and ABI, you will need to specify your time schedule in the form of a [CRON expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm). Once you have entered your CRON expression, click **Next**.
```
Comment (dwightjl) [inline docs/chainlink-keepers/job-scheduler.md:49]:
```suggestion
After you successfully enter your contract address and ABI, specify your time schedule in the form of a [CRON expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm). After you enter your CRON expression, click **Next**.
```

File: docs/chainlink-keepers/job-scheduler.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+![Keepers Cron Expression](/images/contract-devs/keeper/keeper-cron-expression.png)
+
+## Entering Upkeep Details
+
+To complete the upkeep registration process using time-based triggers, you must enter some information about your upkeep includuing its name, gas limit,starting balance, and contact information.
```
Comment (dwightjl) [inline docs/chainlink-keepers/job-scheduler.md:55]:
```suggestion
To complete the upkeep registration process using time-based triggers, you must enter some information about your upkeep including its name, gas limit, starting balance, and contact information.
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/chainlink-keepers/job-scheduler.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+date: Last Modified
+title: 'Chainlink Keepers Job Scheduler'
+whatsnext:
+  {
+    'FAQs': '/docs/chainlink-keepers/faqs/',
+  }
+---
+
+
+Learn how to register a contract with the Chainlink Keepers network and automate it using the Keepers job scheduler. Before you begin, learn how to [create a Keeper-compatible contract](../compatible-contracts).
```
Comment (wentzeld) [inline docs/chainlink-keepers/job-scheduler.md:13]: Please remove this

Before you begin, learn how to [create a Keeper-compatible contract](../compatible-contracts).

When using job scheduler you don't need a Keepers-compatible contract. You do need a deployed contract with a function.

So maybe say

"When using Job Scheduler to automate your contract you just need a deployed contract on chain, it doesn't have to be Keeper-compatible"

File: docs/chainlink-keepers/job-scheduler.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+## Connecting your Wallet
+
+If you do not already have a wallet connected with the Keepers network, the interface will prompt you to do so. Click the **Connect Wallet** button and follow the remaining prompts to connect your wallet to the network.
+
+![Keepers Connect Waller](/images/contract-devs/keeper/keeper-connect-wallet.png)
+
+# Trigger Selection
+
+Once you have successfully connected your wallet, you will have options to create a trigger mechanism for automation. The trigger specifies what Keeper Nodes should look at to determine if your Upkeep should be performed. The page presents two options:
```
Comment (wentzeld) [inline docs/chainlink-keepers/job-scheduler.md:36]: Options to "select" a trigger mechanism. We won't be creating the mechanism.

Remove: "The page presents two options"

File: docs/chainlink-keepers/job-scheduler.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+- [Time-based triggers](#using-time-based-triggers): Uses a time schedule (CRON) to execute your smart contract function according to the schedule
```
Comment (wentzeld) [inline docs/chainlink-keepers/job-scheduler.md:38]: your "deployed smart contract function"

File: docs/chainlink-keepers/job-scheduler.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+# Using Time-Based Triggers
+
+When you select the time-based trigger, you are prompted to enter a *contract address*. This is the address of the deployed contract that contains the function that you want Keepers to execute. If you did not verify the contract on chain, you will also need to paste the [Application Binary Interface](https://docs.soliditylang.org/en/develop/abi-spec.html) (ABI) of the deployed contract into the corresponding text box.
+
+![Keepers Time Based Trigger](/images/contract-devs/keeper/keeper-time-based-trigger.png)
+
+## Specifying the Time Schedule
+
+After you successfully enter your contract address and ABI, specify your time schedule in the form of a [CRON expression](https://docs.oracle.com/cd/E12058_01/doc/doc.1014/e12030/cron_expressions.htm). After you enter your CRON expression, click **Next**.
```
Comment (wentzeld) [inline docs/chainlink-keepers/job-scheduler.md:53]: This link contains values we do not support, let us discuss please

Comment (wentzeld) [reply docs/chainlink-keepers/job-scheduler.md:53]: This is a blocker for me

```
Cron jobs are interpreted according to this format:

  ┌───────────── minute (0 - 59)
  │ ┌───────────── hour (0 - 23)
  │ │ ┌───────────── day of the month (1 - 31)
  │ │ │ ┌───────────── month (1 - 12)
  │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday)
  │ │ │ │ │
  │ │ │ │ │
  │ │ │ │ │
  * * * * *

All times are in UTC

- can be used for range e.g. "0 8-16 * * *"
/ can be used for interval e.g. "0 */2 * * *"
, can be used for list e.g. "0 17 * * 0,2,4"

  Special limitations:
    * there is no year field
    * no special characters: ? L W #
    * lists can have a max length of 26
    * no words like JAN / FEB or MON / TUES
```

Comment (wentzeld) [reply docs/chainlink-keepers/job-scheduler.md:53]: This code block should be in docs please

File: docs/chainlink-keepers/job-scheduler.md (hunk 8 above)
Comment (wentzeld) [reply docs/chainlink-keepers/job-scheduler.md:38]: minor nit, can be ignored

File: docs/chainlink-keepers/job-scheduler.md (hunk 7 above)
Comment (wentzeld) [reply docs/chainlink-keepers/job-scheduler.md:36]: minor nit, can be ignored

File: docs/chainlink-keepers/job-scheduler.md (hunk 9 above)
Comment (ritika-b) [reply docs/chainlink-keepers/job-scheduler.md]: Resolved

File: docs/chainlink-keepers/job-scheduler.md (hunk 8 above)
Comment (ritika-b) [reply docs/chainlink-keepers/job-scheduler.md]: Resolved

File: docs/chainlink-keepers/job-scheduler.md (hunk 7 above)
Comment (ritika-b) [reply docs/chainlink-keepers/job-scheduler.md]: Resolved

**Review by dwightjl — APPROVED** [review]

**Pull request merged**